src/liblog/lib.rs

```diff
@@ -463,49 +460,72 @@ mod tests {
 
     #[test]
     fn no_match() {
-        let dirs = [
-            LogDirective { name: Some("crate2".to_string()), level: 3 },
-            LogDirective { name: Some("crate1::mod1".to_string()), level: 2 }
-        ];
+        let dirs = [LogDirective {
+            name: Some("crate2".to_string()),
+            level: 3,
+        },
+                    LogDirective {
+            name: Some("crate1::mod1".to_string()),
+            level: 2,
+        }];
         assert!(!enabled(2, "crate3", dirs.iter()));
     }
 
     #[test]
     fn match_beginning() {
-        let dirs = [
-            LogDirective { name: Some("crate2".to_string()), level: 3 },
-            LogDirective { name: Some("crate1::mod1".to_string()), level: 2 }
-        ];
+        let dirs = [LogDirective {
+            name: Some("crate2".to_string()),
+            level: 3,
+        },
+                    LogDirective {
+            name: Some("crate1::mod1".to_string()),
+            level: 2,
+        }];
         assert!(enabled(3, "crate2::mod1", dirs.iter()));
     }
 
     #[test]
     fn match_beginning_longest_match() {
-        let dirs = [
-            LogDirective { name: Some("crate2".to_string()), level: 3 },
-            LogDirective { name: Some("crate2::mod".to_string()), level: 4 },
-            LogDirective { name: Some("crate1::mod1".to_string()), level: 2 }
-        ];
+        let dirs = [LogDirective {
+            name: Some("crate2".to_string()),
+            level: 3,
+        },
+                    LogDirective {
+            name: Some("crate2::mod".to_string()),
+            level: 4,
+        },
+                    LogDirective {
+            name: Some("crate1::mod1".to_string()),
+            level: 2,
+        }];
         assert!(enabled(4, "crate2::mod1", dirs.iter()));
         assert!(!enabled(4, "crate2", dirs.iter()));
     }
 
     #[test]
     fn match_default() {
-        let dirs = [
-            LogDirective { name: None, level: 3 },
-            LogDirective { name: Some("crate1::mod1".to_string()), level: 2 }
-        ];
+        let dirs = [LogDirective {
+            name: None,
+            level: 3,
+        },
+                    LogDirective {
+            name: Some("crate1::mod1".to_string()),
+            level: 2,
+        }];
         assert!(enabled(2, "crate1::mod1", dirs.iter()));
         assert!(enabled(3, "crate2::mod2", dirs.iter()));
     }
 
     #[test]
     fn zero_level() {
-        let dirs = [
-            LogDirective { name: None, level: 3 },
-            LogDirective { name: Some("crate1::mod1".to_string()), level: 0 }
-        ];
+        let dirs = [LogDirective {
+            name: None,
```

Comment:
This seems wrong, it should be indented relative to beginning of `LogDirective`.